src/ResourceManager/Network/Commands.Network/help/New-AzureRmExpressRouteCrossConnectionPeering.md

```diff
@@ -0,0 +1,310 @@
+﻿---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+ms.assetid: 5E9C02BE-9DCC-4865-95D2-6B69D373BE77
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermexpressroutecrossconnectionpeering
+schema: 2.0.0
+---
+
+# New-AzureRmExpressRouteCrossConnectionPeering
+
+## SYNOPSIS
+Creates a new peering configuration to be added to an ExpressRoute cross connection.
+
+## SYNTAX
+
+### SetByResource (Default)
+```
+New-AzureRmExpressRouteCrossConnectionPeering -Name <String> -PeeringType <String> -PeerASN <UInt32>
+ -PrimaryPeerAddressPrefix <String> -SecondaryPeerAddressPrefix <String> -VlanId <Int32> [-SharedKey <String>]
+ [-MicrosoftConfigAdvertisedPublicPrefixes <System.Collections.Generic.List`1[System.String]>]
+ [-MicrosoftConfigCustomerAsn <Int32>] [-MicrosoftConfigRoutingRegistryName <String>]
+ [-PeerAddressType <String>] [-LegacyMode <Boolean>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### MicrosoftPeeringConfigRoutFilterId
```

Comment:
Where are these 2 other parameter sets coming from? They are the same as the default.

Reply:
Done